### ISSUE TITLE
Unable to run Quantmod recipes

### ISSUE THREAD

**Issue by zisner** [Row64/Row64_Issue_Tracker#2]
**Row64 Version:** 1.01

**Operating System & Hardware:** Windows 10, NVIDIA GeForce RTX 2080 with Max-Q Design, Intel(R) Core(TM) i7-8750H

**Description:**
Running Recipes that use Quantmod produce a python error.

Here were my steps:
1. Open a new dataframe.
2. Click on Recipes, and select the Finance recipe category.
3. Click **Run** on the Get Symbol recipe.

![image](https://user-images.githubusercontent.com/56556173/150619038-0ddb6f83-1467-454f-aa7f-f64a381dfcf1.png)

Here's the output log:
```
C:\Program Files\Row64\Row64_V1_01\python\Lib\site-packages\urllib3\contrib\socks.py:50: DependencyWarning:

SOCKS support in urllib3 requires the installation of optional dependencies: specifically, PySocks.  For more information, see https://urllib3.readthedocs.io/en/1.26.x/contrib.html#socks-proxies

ERROR:
      6 ch=qm.get_symbol("AAPL", start='09/01/2018')
      7 df=ch.to_frame()
----> 8 ch.iplot()
      9 

PermissionError: [Errno 13] Permission denied: 'temp-plot.html'
```

**Comment by zisner** [comment]
As a current work-around, please run Row64 in administrator mode. We plan to integrate a fix for the next version (1.02).

You can run as administrator by right clicking the Row64 icon and selecting "Run as Administrator".

**Comment by misner5** [comment]
This should be fixed.  Finance recipes have also moved to yahooquery in V3.3
